requirements-selftests.txt

```diff
@@ -2,7 +2,7 @@
 
 # inspektor (static and style checks)
 isort==4.3.21
-pyenchant==2.0.0
+pyenchant==3.0.0
```

Comment:
There is a 3.2.0 version from Dec 8, 2020. Could we use that?

Reply:
yes, sure. It was just a test.